config/locales/hyrax.en.yml

```diff
@@ -1132,17 +1142,21 @@ en:
       defaults:
         admin_set_id: Administrative Set
         based_near: Location
+        contributor: Contributor
```

Comment:
remove line 1145 for this and all other languages

Reply:
Line 1145 contributor: has been removed.